Library/Homebrew/cmd/test-bot.rb

```diff
@@ -425,6 +425,10 @@ def formula formula_name
       install_args = %w[--verbose]
       install_args << "--build-bottle" unless ARGV.include? "--no-bottle"
       install_args << "--HEAD" if ARGV.include? "--HEAD"
+      # Pass --devel or --HEAD to the test-bot in the event formulae lack stable. Supports devel-only/head-only.
+      # head-only should not have devel, but devel-only can have head. Stable can have all three.
+      install_args << "--devel" if formula.devel && !formula.stable?
+      install_args << "--HEAD" if formula.head && !formula.stable? && !formula.devel?
```

Comment:
Yeah, looks like we don't actually currently test the ` head-only ` builds themselves, just audits them and such. There doesn't seem to be any [install stage](http://bot.brew.sh/job/Homebrew%20Head-Only%20Pull%20Requests/lastSuccessfulBuild/version=mavericks/console) in the ` head-only ` testing. I kinda wanted to change that aspect alongside the ` devel-only ` situation, but I'm open to being persuaded it's a terrible idea for ` head-only `.